dlt/common/schema/utils.py

```diff
@@ -684,7 +684,8 @@ def new_table(
         assert schema_contract is None
     else:
         # set write disposition only for root tables
-        table["write_disposition"] = write_disposition or DEFAULT_WRITE_DISPOSITION
+        if write_disposition is not None:
```

Comment:
OK this looks like good place to fix it. but way better would be to use `DEFAULT_WRITE_DISPOSITION` as a default argument value. then it will be backward compatible but also fix the bug

Reply:
Not sure I understand. The task is to drop the default value, but if I put it into the default argument value, it'll be the same as it's now, no?

Reply:
Or you mean the default value for disposition in `apply_hints()`?

Reply:
> Not sure I understand. The task is to drop the default value, but if I put it into the default argument value, it'll be the same as it's now, no?
no, because when argument is skipped (like in 90% of cases) we'll get default values. when we pass None, None will be used. previously when you passed None, default value was used